### DISCUSSION TITLE
How to limit user access to specific applications and roles in my Ant Media Server?

### DISCUSSION THREAD

**Discussion by Mohit-3196** [ant-media/Ant-Media-Server#5220] [FAQs]
I would like to share my Ant Media Server with different clients and I want to know how can I restrict user access to specific applications and roles so that they can only access the application created for them.

**Comment by yashtandon113** [comment]
It is possible since Ant Media Server supports multi-tenancy. You can create users and grant them access to only specified applications by following the documentation below.
https://antmedia.io/docs/get-started/user-management/
